### PR TITLE
fix parse email headers

### PR DESCRIPTION
**Pull request by anara123**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/13217

## Description
ParseEmailHeaders using ParseEmailFiles.
ParseEmailHeaders will be deprecated - instead use ParseEmailFiles

## Screenshots
Paste here any images that will help the reviewer

## Does it break backward compatibility?
   - Yes

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

**Comment by glicht** [comment]
Something wrong with build

**Comment by glicht** [comment]
Also, what about deprecating the headers script?

File: Scripts/ParseEmailFiles/ParseEmailFiles.py [hunk 1]
```diff
@@ -3276,14 +3283,24 @@ def handle_eml(file_path, b64=False, file_name=None):
...
             item_dict = {
                 "name": item[0],
                 "value": convert_to_unicode(item[1])
             }
 
+            # old way to map headers
             header_list.append(item_dict)
 
+            # new way to map headers - dictionary
+            headers_map[item[0]] = convert_to_unicode(item[1])
```
Comment (glicht) [inline Scripts/ParseEmailFiles/ParseEmailFiles.py]: Need to think this through. We may be overriding an existing header when an email contains multiple headers of the same name.

Comment (anara123) [reply Scripts/ParseEmailFiles/ParseEmailFiles.py]: fixed

**Comment by anara123** [comment]
fixes https://github.com/demisto/etc/issues/14440

**Pull request merged**